slam-ws/src/nav_goal/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(nav_goal)
+
+find_package(catkin REQUIRED COMPONENTS
```

Comment:
Maybe we should keep comments for sections that we keep?
```
## Find catkin macros and libraries
## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
## is used, also find other catkin packages
```

Reply:
Seems like a good idea, will fix this. Should I fix for nav_husky_gazebo as well?

Reply:
Yes please